sema/commons/yml/loader.py

```
@@ -0,0 +1,42 @@
+from logging import Logger, getLogger
+
+from yaml import Loader, SafeLoader
+from yaml.nodes import ScalarNode
+
+log: Logger = getLogger(__name__)
+
+
+class LoaderBuilder:
+    def __init__(self):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)